bin/fba_sv1

```diff
@@ -78,24 +78,78 @@ def get_isaen(g, aen):
     )
 
 
-# AR get the updated ra,dec with proper motion
-def get_nowradec(ra, dec, pmra, pmdec, parallax, ref_epoch, new_obstime):
+# AR courtesy of DL : from astrometry.net's util/starutil_numpy.py
+def datetomjd(d):
+    d0 = datetime(1858, 11, 17, 0, 0, 0)
+    dt = d - d0
+    return dt.days + (dt.seconds + dt.microseconds / 1e6) / 86400.0
+
+
+# AR courtesy of DL : adapted from legacypipe.survey
+def mjd_to_year(mjd):
+    # mjd_to_year(57205.875) -> 2015.5
+    daysperyear = 365.25  # Julian years, by definition
+    J2000 = datetime(2000, 1, 1, 12, 0, 0, 0, tzinfo=None)
+    mjd2k = datetomjd(J2000)
+    return (mjd - mjd2k) / daysperyear + 2000.0
+
+
+# AR courtesy of DL : from legacypipe.survey
+# AR originally named "radec_at_mjd()", renamed to get_nowradec
+def get_nowradec(ra, dec, pmra, pmdec, parallax, ref_year, new_obstime):
+    # AR new_obstime = Time.now()
+    """
+    Units:
+    - matches Gaia DR1/DR2
+    - pmra,pmdec are in mas/yr.
+      pmra is in angular speed (ie, has a cos(dec) factor)
+    - parallax is in mas.
+    Returns: RA,Dec
+    """
+    mjd = new_obstime.mjd
     # AR new_obstime = Time.now()
     # AR clipping Gaia PARALLAX to >1e-3 (setting distance at <1Mpc)
     parallax_clip = parallax.copy()
     parallax_clip[(~np.isfinite(parallax)) | (parallax < 1e-3)] = 1e-3
```

Comment:
We should discuss on list if this is still desired.  This was necessary when I thought that apply_space_motion was going to be the logical route, since a negative distance doesn't make a lot of sense.  But a negative parallax is fine in the context of this code.  We are going to be inserting a small noise bias if we clip the parallaxes in this way, so I would probably vote for deleting the clipping now.

Reply:
I agree with you, I ve currently removed the parallax clipping